src/main/java/seedu/address/logic/parser/AddConcertCommandParser.java

```diff
@@ -47,8 +50,8 @@ public AddConcertCommand parse(String args) throws ParseException {
     }
```

Comment:
Can consider adding assertions for isValid for the fields passed to the constructor of Concert similar to `AddPersonCommandParser`